src/client/client4.js

```diff
@@ -2552,9 +2559,9 @@ export default class Client4 {
         });
     };
 
-    trackEvent(category, event, props) {
-        // Temporary change to only track one event to reduce data rate - see MM-13062
-        if (event !== 'api_posts_create') {
+    trackEvent(category, event, props, allow = false) {
+        // Temporary change to allow only certain events to reduce data rate - see MM-13062
+        if (!allow) {
```

Comment:
I'd prefer we have an explicit whitelist in this function, because otherwise it's going to be annoying to track down what's whitelisted/not whitelisted by having to look all over the code in this and the web/mobile repos.

Also, did @jasonblais review this and sign off on tracking these events?

Reply:
Yes, I signed off on this one.

Reply:
Makes sense, I'll make it more explicit